### ISSUE TITLE
[BUG] Minor, badge rendering with README file

### ISSUE THREAD

**Issue by fakoua** [nestlandofficial/nest.land#132]
The readme file renders the badges in the same line with github, while nest renders the badge in two separate lines, minor but fixing it would be great.
Thanks

**Comment by tbaumer22** [comment]
@fakoua thanks for letting us know. Could you attach a link and/or a screenshot so we can see what is happening?

**Comment by fakoua** [comment]
> @fakoua thanks for letting us know. Could you attach a link and/or a screenshot so we can see what is happening?

For example:

https://github.com/fakoua/deno_gui 
and
https://nest.land/package/deno_gui

Please check how the badge are rendered.

Thanks

**Comment by tbaumer22** [comment]
I see. This is very interesting. If you visit [Opine's module page](https://nest.land/package/opine/files/README.md), this does not occur. As a workaround, try formatting your README badges like Craig Morten does here: https://raw.githubusercontent.com/asos-craigmorten/opine/main/README.md

Let me know if this helps fix your problem!

**Comment by fakoua** [comment]
Hi @tbaumer22 , 
It is fine for Opine, but my concern is not a specific package, we need consistency between github and nest.

**Comment by tbaumer22** [comment]
Yes, you're right. I completely agree. We'll investigate a solution to this problem when we can. Until then, feel free to use Craig Morten's method above!

Thanks again for reporting this.

**Comment by maximousblk** [comment]
seems to be a problem with `vue-markdown`. it adds a `<br>` tag after each image.

![image](https://user-images.githubusercontent.com/28438021/85946695-df735580-b963-11ea-86b5-99de579a3ce9.png)

**Comment by balupton** [comment]
You can use https://github.com/bevry/projectz for maintaing the readme badges - it uses `package.json:badges` and other information from inside `package.json` to populate them.

**Comment by t8** [comment]
I'm afraid that our issue here has to do with the way the badges are displayed on the site and not the badge files themselves.

@MartonDev is this something that can be fixed with your experimental `vue-markdown` in development?
